pkg/util/kernel/fs.go

```diff
@@ -59,6 +59,19 @@ var SysFSRoot = funcs.MemoizeNoError(func() string {
 	return "/sys"
 })
 
+// BootRoot retrieves the current boot dir we should use
+var BootRoot = funcs.MemoizeNoError(func() string {
+	if v := os.Getenv("HOST_BOOT"); v != "" {
+		return v
+	}
+	if os.Getenv("DOCKER_DD_AGENT") != "" {
```

Comment:
`env.IsContainerized` is doing the same thing, but it more cleanly expresses the goal